Reflections/Week_2_Reflection.md

```diff
@@ -0,0 +1,9 @@
+**Will Pratt**
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+* I have begun adding comments and have gone about creating states for rooms and checking them to determine what should be done depending on the room.
```

Comment:
Great progress on getting states working so far. I would like to point to [this part](https://adventurelib.readthedocs.io/en/stable/rooms.html#storing-attributes-on-rooms) of the AdventureLib documentation for special consideration.